libs/auth/scripts/src/generate_dev_keys_and_certs.ts

```diff
@@ -123,8 +103,11 @@ async function generateDevKeysAndCerts({
   const vxCertAuthorityCertPath = `${outputDir}/vx-cert-authority-cert.pem`;
   const vxPrivateKey = await generatePrivateKey();
   await fs.writeFile(vxPrivateKeyPath, vxPrivateKey);
-  const vxCertAuthorityCert =
-    await generateDevVxCertAuthorityCert(vxPrivateKeyPath);
+  const vxCertAuthorityCert = await generateSelfSignedCert({
+    privateKeyPath: vxPrivateKeyPath,
+    commonName: 'VotingWorks Development',
+    expiryDays: CERT_EXPIRY_IN_DAYS.DEV,
+  });
```

Comment:
As noted in the PR description:

> In a subsequent PR, I'll commit an updated dev root cert with a "VotingWorks Development" common name added. That's generated using the `generate-dev-keys-and-certs` script, also updated in this PR. I'm committing that separately because that entails updating CVR fixtures (to generate a new .vxsig file) which results in a pretty big diff that bogs the GitHub UI down.